### DISCUSSION TITLE
Does LME require a Linux server?

### DISCUSSION THREAD

**Discussion by cfwillis** [cisagov/LME#213] [FAQ] [answered]
I don't understand if LME requires a real (not a vm) Linux server, or does the docker container discussed in the documentation make it so you can setup and run a Linux server inside a Windows server, and only need to utilize real Windows servers?

**Comment by cbaxley** [comment, marked as answer]
If you look in the testing folder, there are docker builds that allow you
to run it in a container.
I was just imagining that you could run it on your Windows server if you
wanted to.
You would have to make sure that the docker container got its IP from the
DHCP of the
network and that you made your network dhcp give it a static ip.
I am definitely running it two different scenarios:
1. Wsl Ubuntu direct install
2. Docker in docker using Wsl
It is unsupported, but It does run and you would just have to map the ip
and the dns name
correctly.

Thanks,
Clint Baxley
***@***.***

Windows is great... I used it to download Linux.

**Reply by cfwillis** [reply]
Thank you for this answer. We are a very small two person outfit with 6 Windows servers in a AD setup to protect. It looked like we were a perfect fit for LME, until I got to the part where it seemed like we had to add a Linux server to our network just for LME, which gave me pause as we have only a Windows skillset at the moment. We know how to protect and configure Windows servers but not Linux. We will look into Docker using Wsl.

If anyone knows of a good commercial product with basic SIEM features that is easy enough for 1 or 2 people to manage a Windows **only** server network, please share.
